command/catalog_command.go

```diff
@@ -0,0 +1,50 @@
+package command
+
+import (
+	"strings"
+
+	"github.com/mitchellh/cli"
+)
+
+var _ cli.Command = (*CatalogCommand)(nil)
+
+type CatalogCommand struct {
+	BaseCommand
+}
+
+func (c *CatalogCommand) Run(args []string) int {
+	return cli.RunResultHelp
+}
+
+func (c *CatalogCommand) Help() string {
+	helpText := `
+Usage: consul catalog <subcommand> [options] [args]
+
+  This command has subcommands for interacting with Consul's catalog. The
+  catalog should not be confused with the agent, although the APIs and
+  responses may be similar.
+
+  Here are some simple examples, and more detailed examples are available
+  in the subcommands or the documentation.
+
+  List all datacenters:
+
+      $ consul catalog datacenters
```

Comment:
Yea, that was my fear. I just also fear that we are going too deep (queue Inception gif). I find CLIs challenging to reason about beyond the third level, which is why I like the kv CLI

```
consul kv [action]
```

That's why I suggested 

```
consul catalog list-datacenters
```

since then list is a verb. Otherwise, we have inconsistency where some CLI commands are "namespace verb" and others are "namespace noun verb".